CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 FEATURES:
 
 * `vcd_vapp_vm` - Ability to add metadata to a VM. For previous behaviour please see `BACKWARDS INCOMPATIBILITIES` ([#158](https://github.com/terraform-providers/terraform-provider-vcd/issues/158))
+* `vcd_snat` and `vcd_dnat` - now supports organization network name. For previous behaviour please see `BACKWARDS INCOMPATIBILITIES` 
```

Comment:
This is not "support".
We're changing the API in an incompatible way. If we apply this change, we need to increase the major version number (go to 3.x) because existing configurations will be broken.

Reply:
Well regard back compatibility - What are options? Change to optional? even though current implementation kind'a is one big bug as it works only as expected if you have one organization. What are everyone's ideas?

Reply:
Two options:
1. Deprecate the current resource, leaving it as it is, and create a new one with the mandatory network. property.
2. Add the property as optional, and default to the first network if no property is provided

Reply:
Absolutely, this new option should be optional and the current behaviour kept in tact. We can't break all the configurations out there.

Reply:
BTW, is there any way to show deprecation warning when configuration will **not** be using this new `network_name` field? This way we could warn the users in 2.2.0 and then make the field required later.

Reply:
Yes. We can add the field `DeprecationMessage` to the resource.

Reply:
Done. Change to optional, added deprecation message, update documentation